### ISSUE TITLE
Will bitlbee-discord work with a big account?

### ISSUE THREAD

**Issue by akfaew** [sm00th/bitlbee-discord#231]
When I connected a small account which was a member of one small server everything worked fine.

But when I connected an account that's a member of over 20 servers that have over 900 rooms (and tens of thousands of users) it stopped working. It could list all rooms correctly, and (very slowly) I was able to join all rooms. But they never "started to work", it was always me and @root in the channels, they never loaded.

Are there perhaps any settings I could toggle to make it work with a bigger account? Can I disable some functionality that's causing it to be sluggish?

**Comment by sm00th** [comment]
Yes, bigger accounts are known to have issues and also make it impossible to meaningfully follow anything so there are two options to work around this: `auto_join` which allows you to not join any channels automatically and manually add the ones you are interested in and `auto_join_exclude` which allows join everything but specified patterns. Both are covered in README (last part of `usage` section also explains how to add channels manually). For accounts with a lot of servers first option usually works better.

**Comment by akfaew** [comment]
I understand their proprietary API is rather crappy. Just to confirm, it's currently not possible to reliably be part of the 900 rooms?

**Comment by sm00th** [comment]
It is more complicated than just the API, but yes, bitlbee-discord can't do 900 rooms properly.
